### ISSUE TITLE
Etag doesn't change

### ISSUE THREAD

**Issue by thisIsTheFoxe** [nextcloud/files_lock#57]
> locking/unlocking a file must change its eTag (and get propagated up to root), as otherwise clients will not reload the folder content and thus are not aware of locked state

_Originally posted by @tobiasKaminsky in https://github.com/nextcloud/files_lock/issues/24#issuecomment-736596674_

I found this while debugging. I assume currently locking a file doesn't change its etag?

```
(lldb) p metadata.fileName
(String) $R0 = "index.html"
(lldb) p metadata.fileName
(String) $R1 = "scan 22-03-15 10-37-49.pdf"
(lldb) p metadata.etag
(String) $R2 = "42e56559fc5322e74d67f60efe8198ba"
(lldb) p result.etag
(String) $R3 = "42e56559fc5322e74d67f60efe8198ba"
(lldb) p result.lock
(Bool) $R4 = false
(lldb) p metadata.lock
(Bool) $R5 = true
```

**Comment by juliushaertl** [comment]
Yes, there is currently no ETag change, but should be rather straight forward to propagate a change on lock/unlock. I'll have a look.

**Comment by juliushaertl** [comment]
Implemented with [3ee9e47](https://github.com/nextcloud/files_lock/pull/54/commits/3ee9e47b76032d15d88d53845cd346a95ed6c791) in https://github.com/nextcloud/files_lock/pull/54
